setup.py

```diff
@@ -34,7 +34,7 @@ def run(self):
 
 
 install_requires = [
-    "tentaclio",
+    "tentaclio==1.0.3",
```

Comment:
could add sqlalchemy explicitly here and and put the <1.4 dependency there to add a little more wiggle room

Reply:
Yeah makes sense